CREATE_BOT.md

```diff
@@ -0,0 +1,36 @@
+# Creating a Bot Account
+In order to work with the Music Bot and the Discord API in general, we must first create a Discord Bot account.
+
+Creating a Bot account is pretty straightforward process.
+
+1. Make sure you're logged on to the [Discord website](https://discord.com/)
+2. Navigate to the [application page](https://discord.com/developers/applications)
+3. Click on the “New Application” button.
+4. Give the application a name and click “Create”.
+5. Create a Bot User by navigating to the “Bot” tab and clicking “Add Bot”.
+    * Click “Yes, do it!” to continue.
+6. Make sure that Public Bot is ticked if you want others to invite your bot.
+    * You should also make sure that Require OAuth2 Code Grant is unchecked unless you are developing a service that needs it. If you’re unsure, then leave it unchecked.
+7. Copy the token using the “Copy” button and paste it in your "config.json" file.
+    * This is not the Client Secret at the General Information page. 
+    * Warning: It should be worth noting that this token is essentially your bot’s password. You should never share this to someone else. In doing so, someone can log in to your bot and do malicious things, such as leaving servers, ban all members inside a server, or pinging everyone maliciously.
+
+And that’s it. You now have a bot account and you can login with that token.
+
+# Inviting Your Bot
+
+So you’ve made a Bot User but it’s not actually in any server.
+
+If you want to invite your bot you must create an invite URL for it.
+
+1. Make sure you're logged on to the [Discord website](https://discord.com/)
+2. Navigate to the [application page](https://discord.com/developers/applications)
+3. Click on your bot's page.
+4. Go to the "OAuth2" tab.
+5. Tick the “bot” checkbox under “scopes”.
+6. Tick the permissions required for your bot to function under “Bot Permissions”.
```

Comment:
Do mention the required permissions required for this particular bot that is connect, speak, send message, view channels, add reaction (for future use)